h2o-automl/src/main/java/ai/h2o/automl/modeling/XGBoostSteps.java

```diff
@@ -0,0 +1,188 @@
+package ai.h2o.automl.modeling;
```

Comment:
nothing new here: just extracted this class for decoupling (it contains dependencies to XGBoost module) as it used to be internal to `XGBoostStepsProvider`